web/src/main/java/org/bookkeepervisualmanager/api/resources/SystemStatusResource.java

```diff
@@ -0,0 +1,191 @@
+            this.lostBookieRecoveryDelay = clusterWideConfiguration.getLostBookieRecoveryDelay();
+            this.layoutFormatVersion = clusterWideConfiguration.getLayoutFormatVersion();
+            this.layoutManagerFactoryClass = clusterWideConfiguration.getLayoutManagerFactoryClass();
+            this.layoutManagerVersion = clusterWideConfiguration.getLayoutManagerVersion();
+        }
+
+        public String getAuditor() {
+            return auditor;
+        }
+
+        public boolean isAutorecoveryEnabled() {
+            return autorecoveryEnabled;
+        }
+
+        public int getLostBookieRecoveryDelay() {
+            return lostBookieRecoveryDelay;
+        }
+
+        public int getLayoutFormatVersion() {
+            return layoutFormatVersion;
+        }
+
+        public String getLayoutManagerFactoryClass() {
+            return layoutManagerFactoryClass;
+        }
+
+        public int getLayoutManagerVersion() {
+            return layoutManagerVersion;
+        }
+
+        public String getBookkeeperConfiguration() {
+            return bookkeeperConfiguration;
+        }
+
+        public void setBookkeeperConfiguration(String bookkeeperConfiguration) {
+            this.bookkeeperConfiguration = bookkeeperConfiguration;
+        }
+
+        public long getLastCacheRefresh() {
+            return lastCacheRefresh;
+        }
+
+        public void setLastCacheRefresh(long lastCacheRefresh) {
+            this.lastCacheRefresh = lastCacheRefresh;
+        }
+
+        public String getStatus() {
+            return status;
+        }
+
+        public void setStatus(String status) {
+            this.status = status;
+        }
+
+    }
+
+    @GET
+    @Secured
+    @Path("info")
+    @Produces(MediaType.APPLICATION_JSON)
+    public SystemStatus getInfo() throws Exception {
+        return new SystemStatus(getBookkeeperManger().getRefreshWorkerStatus());
+    }
+
+    @GET
+    @Secured
+    @Path("clusterconfiguration")
+    @Produces(MediaType.APPLICATION_JSON)
+    public ClusterWideConfiguration getClusterWideConfiguration() throws Exception {
+        BookkeeperManager.RefreshCacheWorkerStatus refreshWorkerStatus = getBookkeeperManger().getRefreshWorkerStatus();
+        BookkeeperManager.ClusterWideConfiguration clusterWideConfiguration = refreshWorkerStatus.getLastClusterWideConfiguration();
+        return new ClusterWideConfiguration(clusterWideConfiguration.getAuditor(),
+                clusterWideConfiguration.isAutorecoveryEnabled(),
+                clusterWideConfiguration.getLostBookieRecoveryDelay(),
+                clusterWideConfiguration.getLayoutFormatVersion(),
+                clusterWideConfiguration.getLayoutManagerFactoryClass(),
+                clusterWideConfiguration.getLayoutManagerVersion(),
+                refreshWorkerStatus.getBookkkeeperClientConfiguration());
+    }
+
+    public static final class ClusterWideConfiguration {
```

Comment:
Maybe it's better in a separate class this time?

Reply:
this class is not used. I have dropped it.
I have moved LedgerBean to a separate class